### PR TITLE
docs: specify date format for reviewed tag

### PR DESCRIPTION
**Pull request by yallen011**
specify a date format for the reviewed tag and provided an example.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by yallen011** [comment]
does the`{@a review-criteria}` part needs to be removed.

**Comment by aikidave** [comment]
> does the`{@a review-criteria}` part needs to be removed.

It does no harm, even if we aren't using the anchor tag.

**Review by aikithoughts — APPROVED** [review]
Looks good to me. Added the `aio: preview` label so we can see the rendered content.

Reviewed-for: global-docs-approvers

File: aio/content/guide/reviewing-content.md [hunk 1]
```diff
@@ -26,7 +26,9 @@ You can review a topic using either the GitHub user interface or in an editor on
 
 1. Locate the last reviewed date at the bottom of the topic and verify that the topic meets the [review criteria](#review-criteria).
 
-   If the topic does not have a last reviewed date, you are welcome to add it to the topic.
+   If the topic does not have a last reviewed date, you are welcome to add it to the topic. To add a date, use the `YYYY-MM-DD` date format. Example: 
+   `@reviewed 2021-03-23`
+
```
Comment (gkalpak) [inline aio/content/guide/reviewing-content.md:31]:
```suggestion
```

**Comment by yallen011** [comment]
@aikidave I think it can be removed because it currently looks incorrect syntax that didn't get parsed and all the headers are linked without it there

**Comment by gkalpak** [comment]
> does the `{@a review-criteria}` part needs to be removed.

We do use it [here][1], so we should not remove it. The fact that the heading-based auto-generated anchor is the same (and thus the link works even if we remove the `@a` anchor is coincidental and could change in the future. You can find more info on why we prefer `@a` anchors [here][2].

[1]: https://github.com/angular/angular/blame/abf99f300e86d88d09aeff39f7d8c3983253a63d/aio/content/guide/reviewing-content.md#L27
[2]: https://angular.io/guide/docs-style-guide#section-header-anchors

**Review by gkalpak — CHANGES_REQUESTED** [review]
Good catch! Thx, @yallen011 :+1:

BTW, CI is unhappy, because the commit message does not follow our [commit message guidelines](https://github.com/angular/angular/blob/655507012b7b59c83e27429349ce3ac10608a82d/CONTRIBUTING.md#commit) 😞
You can make CI happy by changing the commit message to something like:

> docs: specify date format for \`@reviewed\` tag
>
> Specify a date format for the \`@reviewed\` tag and provide an example.

File: aio/content/guide/reviewing-content.md [hunk 2]
```diff
@@ -26,15 +26,16 @@ You can review a topic using either the GitHub user interface or in an editor on
...
+   `@reviewed 2021-03-23`
+   
 
 1. Read through the topic.
 
 1. If the topic requires an update, either [file an issue](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-submitting-an-issue) that describes the update required, or [create a pull request](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-submitting-an-issue) with the update.
 
 1. Update the `@reviewed` tag, either through the [GitHub user interface](guide/updating-content-github-ui) or through Angular's [standard pull request process](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-submitting-an-issue).
 
-{@a review-criteria}
```
Comment (gkalpak) [inline aio/content/guide/reviewing-content.md:37]: As mentioned in https://github.com/angular/angular/pull/41269#issuecomment-802267819, we should rather keep this.

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/guide/reviewing-content.md (hunk 2 above)
Comment (yallen011) [reply aio/content/guide/reviewing-content.md:37]: so keep it as `-{@a review-criteria}` or just `{@a review-criteria}`

Comment (aikidave) [reply aio/content/guide/reviewing-content.md:37]: It should be:

`{@a review-criteria}`

Comment (yallen011) [reply aio/content/guide/reviewing-content.md:37]: ok, will change it now

Comment (yallen011) [reply aio/content/guide/reviewing-content.md:37]: changed

Comment (gkalpak) [reply aio/content/guide/reviewing-content.md:37]: Did you forget to push the changes? I am still seeing the old commit 😕

Comment (yallen011) [reply aio/content/guide/reviewing-content.md]: @gkalpak sorry, added it. I thought it got pushed last time but it didn't

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**